lib/kitchen/driver/azurerm.rb

```diff
@@ -253,7 +251,7 @@ def create(state)
         end
 
         if config[:nic_name].to_s == ""
-          vmnic = "nic-#{config[:vm_name]}"
```

Comment:
This seemed wrong since everything else is referencing `state[:vm_name]` so I just updated this for consistency.